src/aks-preview/azext_aks_preview/tests/latest/test_managed_cluster_decorator.py

```diff
@@ -5924,7 +5924,7 @@ def test_update_workload_identity_profile__default_value_mc_enabled(self):
         )
         dec.context.attach_mc(mc)
         updated_mc = dec.update_workload_identity_profile(mc)
-        self.assertIsNone(updated_mc.security_profile.workload_identity)
+        self.assertIsNotNone(updated_mc.security_profile.workload_identity)
```

Comment:
this seems to be a behavior change, can you explain it?

Reply:
> this seems to be a behavior change, can you explain it?

Currently `update_workload_identity_profile`'s behaviors differ between GA and Preview.

And this test fix meets stable cli's expectation.